scenario/__init__.py

```diff
@@ -6,6 +6,7 @@
     Action,
     Address,
     BindAddress,
+    CloudSpec,
```

Comment:
Per comment elsewhere, we should add `CloudCredential` here as well -- all the state objects should be accessible as `scenario.Foo`.